### PR TITLE
Add polymorphism support for python

### PR DESCRIPTION
**Pull request by woz5999**
Previous PR #6066 left out changes to api_client

### CODE REVIEW

**Comment by woz5999** [comment]
circleci build failed with a connection error communicating with maven. can someone re-run that build?

**Comment by woz5999** [comment]
travis build failed for exceeding github api rate limits:
[ERR!] GitHub API: GitHub rate limit reached.

File: modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/PythonClientCodegen.java [hunk 1]
```diff
@@ -41,7 +41,8 @@ public PythonClientCodegen() {
         // clear import mapping (from default generator) as python does not use it
         // at the moment
         importMapping.clear();
-
+
```
Comment (wing328) [inline modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/PythonClientCodegen.java]: @woz5999 please remove the leading spaces

**Comment by wing328** [comment]
@woz5999 the Github rate limit issue should be addressed in the latest master. For this PR, I'll later test manually to ensure the test result is good.

**Comment by wing328** [comment]
Python tests failed with the following errors:
```
======================================================================
FAIL: deserialize dict(str, Enum_Test)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/travis/build/swagger-api/swagger-codegen/samples/client/petstore/python/tests/test_deserialization.py", line 41, in test_enum_test
    outer_enum=petstore_api.OuterEnum.PLACED))
AssertionError: {'enu[22 chars]number': 1.1,
 'enum_string': 'UPPER',
 'outer_enum': {}} != {'enu[22 chars]number': 1.1,
 'enum_string': 'UPPER',
 'outer_enum': 'placed'}
```

**Comment by woz5999** [comment]
Pushed some changes. Tests are now passing locally.

**Comment by wing328** [comment]
Tests passed via https://travis-ci.org/swagger-api/swagger-codegen/builds/260103233

**Pull request merged**